rfcs/text/0000-tech-planning-process.md

```diff
@@ -0,0 +1,136 @@
+| **M2W4 (Week 8)**  | - A 1 hour TL meeting (the TL Coalescing meeting) MUST occur to consolidate the objectives that have been individually presented and any projects involving multiple components will be sized. |
+|                    | - After the objectives have been coalesced, the union of all presented objectives (minus any objectives that were removed due to low priority) will be sent out to TLs to individually rank in priority. |
+| **M3W1 (Week 9)**  | - A second 1 hour TL meeting (the TL Prioritizing meeting) will take place to bring together all TLs rankings to narrow down the objectives list to objectives that are high priority and feasible. |
+| **M3W2 (Week 10)** | - The TL Planning Shepherd will turn the objectives agreed upon as representative of the TL priorities in the TL Prioritization meeting into an Information RFC and publish. This represents the final artifact: the **Technical Objectives RFC**. There will be no review period as the key stakeholders (the TLs) have already agreed upon the list. |
+| **M3W3 (Week 11)** | - A 2 hour meeting MUST occur with TLs and PMs “to review any incomplete Objectives in the Release, the next Objectives in the DCP Roadmap, and the ***Technical Objectives RFC***.” |
+
+**Note: MXWY is equal to “week Y in milestone X.”
+
+**Note: this schedule may differ based on the holiday schedule each quarter. The responsibility is on the Shepherd of
+the quarter to make sure the timeline is clear (especially the date of the TL+PM call) prior to the process commencing.
+
+### Templates
+
+For the purpose of curating the individual component objectives and for the consolidation of objectives into one that
+represents all the components’ objectives, please use: 
+[TL Objectives Template](https://docs.google.com/document/d/1jUo9Y7y9BJhVe8qpgljI45W7JpNlq3ownVBLA3wK0Qw/edit).
+
+Once the consolidation has taken place, as well as the TL meeting in Week 9, the TL Roadmapping Shepherd MUST turn the
+document into an Informational RFC and publish. The template for crafting an RFC for this purpose is 
+[here](https://github.com/HumanCellAtlas/dcp-community/blob/4bb18c65af5b244460cf404465b1dbc4ee0cc65e/rfcs/roadmap-template.md) 
+but should be tailored such that Themes are removed and replaced with the technical priorities.
+
+### Specific Shepherd Tasks
+
+1) Find out the date/week of the TL+PM planning call and post a timeline on when artifacts are expected to be completed
+to Slack **#tech-architecture** so that folks are aware that the planning process has begun and expectations are set.
+
+2) Schedule a 1 hour TL Coalescing Meeting and a 1 hour TL Prioritization Meeting, including all TLs\*, one and two
+weeks after the individual components have been assigned to start working on their individual components’ objectives,
+respectively.
+
+3) If the meetings aren't separately scheduled, by default it will occur during the two consecutive Tech Architecture
```

Comment:
If it's not during the TA slot, then I think both meetings should be organized at least 2 weeks in advance of the first one, so people aren't caught out be scheduling conflicts.

Reply:
Added a note!